cfgov/v1/__init__.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import  # Python 2 only
```

Comment:
What's the implication of the `# Python 2 only` code comment for future development? Will this be removed when going to Python 3 or whatever?